src/test/ui/consts/const-eval/const_transmute.rs

```diff
@@ -1,4 +1,3 @@
-// compile-pass
 // run-pass
```

Comment:
Could we make situations like these where we have conflicting instructions to be an error?

Reply:
(You already did)